src/datetime/src/components/TimePicker.js

```diff
@@ -3,9 +3,61 @@ import { mergeDeep } from 'immutable';
 import { TimePicker } from '@atlaskit/datetime-picker';
 import selectStyles from '../../../style/selectStyles';
 
+const times = [
```

Comment:
Did you consider adding an extra prop to let users choose if they want the half hours in addition to the whole hours? Also, what about am/pm?

Reply:
Hmm I didn't consider that actually. I know for CWP having half hours is very necessary, so that's all I was going off of. I can add the prop though if it'll be useful.

The times are converted to AM/PM when displayed in the component! But then stored as 24-hour times.

Reply:
Got it. No change necessary right now, but I think it'd be cool if it was possible to let users chose the granularity of the minutes. For example, they can specify like `1` for every minute, or `10` for every 10 minutes, or `15` for every 15 minutes, or `30` for every half hour, and the component will auto-generate the options based off of that.

Reply:
Woah ya, that'd be pretty cool! That's a lot more flexible and less clunky than having a giant array that just sits in a constants file.

Reply:
@UnsungHero97 You can still input whatever time you want in between. this is just to provide a bigger set of common values. AM/PM is not necessary as the labels convert the 24 hour time to show them.

![image](https://user-images.githubusercontent.com/27182199/65716854-cba56680-e054-11e9-8455-cf2bcc191714.png)